src/Sfa.Tl.ResultsAndCertification.Web/Controllers/AdminDashboardController.cs

```diff
@@ -223,19 +225,23 @@ public async Task<IActionResult> ReviewChangeStartYearAsync(int pathwayId)
         [Route("admin/submit-review-changes-start-year", Name = RouteConstants.SubmitReviewChangeStartYear)]
         public async Task<IActionResult> ReviewChangeStartYearAsync(ReviewChangeStartYearViewModel model)
         {
```

Comment:
Model validation? Do we need before it's saved?

Reply:
this is already been checked when submitting from previous page